shoop/core/models/suppliers.py

```diff
@@ -33,7 +34,7 @@ class Supplier(ModuleInterface, models.Model):
     module_data = JSONField(blank=True, null=True)
 
     def __str__(self):
-        return self.name
+        return ascii_safe_repr(self.name)
```

Comment:
I think this is wrong place for the fix and the fix is wrong.

Since the error you're trying to fix happens when ModuleInterface._load_module tries to do repr for the module, IMHO the correct fix is to use ascii_safe_repr there (i.e. in shoop/core/modules/interface.py).

Also str != repr: Don't make string presentation of the module be ascii only, since that is shown in Admin UIs etc. Instead, if you want to make the repr behave nicely, derive the Supplier from ShoopModel rather than models.Model. (`from ._base import ShoopModel`)